docs/aws/README.md

```diff
@@ -28,7 +28,7 @@ This command will create IAM resources used by Karpenter. We recommend using [Cl
 ```bash
 aws cloudformation deploy \
   --stack-name Karpenter-${CLUSTER_NAME} \
-  --template-file ./docs/aws/karpenter.cloudformation.yaml \
+  --template-file $(pwd)/$(curl -fsSO https://raw.githubusercontent.com/awslabs/karpenter/v0.2.4/docs/aws/karpenter.cloudformation.yaml | echo karpenter.cloudformation.yaml)  \
```

Comment:
Would it be possible to wire this up with `mktemp -d` to avoid littering the pwd?